lib/evse_security.cpp

```diff
@@ -211,11 +216,13 @@ DeleteCertificateResult EvseSecurity::delete_certificate(const CertificateHashDa
     bool found_certificate = false;
     bool failed_to_write = false;
 
+    // TODO (ioan): load all the bundles since if it's the V2G root in that case we might have to delete
+    // whole hierarchies
     for (auto const& [certificate_type, ca_bundle_path] : ca_bundle_path_map) {
         try {
             X509CertificateBundle ca_bundle(ca_bundle_path, EncodingFormat::PEM);
 
-            if (ca_bundle.delete_certificate(certificate_hash_data)) {
+            if (ca_bundle.delete_certificate(certificate_hash_data, true)) {
```

Comment:
If we want to delete the children of a root certificate, couldn't those be in another bundle?
Deleting the children of a deleted certificate isn't required in OCPP 2.0.1, so this is probably not critical, but I'm still asking because it seems to be the intention here.